examples/classification/configs/pruning/resnet18_pruning_magnitude.json

```diff
@@ -26,9 +26,9 @@
    "compression": [
        {
             "algorithm": "filter_pruning",
+            "pruning_init": 0.1,
```

Comment:
I didn't get why it should be so.

I have all the cofigs looking as follows:
```
"compression": [
       {
          "algorithm": "filter_pruning",
          "params": {
              "schedule": "exponential",
              "pruning_init": 0.1,
              "pruning_target": 0.4,
              "pruning_steps": 15,
              "weight_importance": "geometric_median" / "L2"
          }
        }
     ]
```

and they work fine!

Reply:
@AlexKoff88   In PR #231 we moved `pruning_init` to the logic of the algorithm.